packages/compile-solidity/test/sources/v0.8.x/Version7Pragma.sol

```diff
@@ -0,0 +1,9 @@
+pragma solidity ^0.8.0;
+
+contract Version7Pragma {
```

Comment:
Should this be `Version8Pragma` along with a file rename?

Reply:
Oops! Thank you.